src/index.js

```diff
@@ -30,24 +30,16 @@ const Container = ({ options }) =>
 const onfidoRender = (options, el, merge) =>
   render( <Container options={options}/>, el, merge)
 
-const complete = (options) => {
-  return () => {
-    Tracker.sendEvent('completed flow')
-    options.onComplete()
-  }
-}
-
-const bindCompleteEvent = (options) => {
-  events.on('complete', complete(options))
-}
+const trackOnComplete = () => Tracker.sendEvent('completed flow')
+events.on('complete', trackOnComplete)
 
-const unbindCompleteEvent = (options) => {
-  events.off('complete', complete(options))
+const bindOnComplete = ({onComplete}) => {
+  events.on('complete', onComplete)
 }
 
-const rebindCompleteEvent = (newOptions, bindedComplete) => {
-  unbindCompleteEvent(bindedComplete)
-  bindCompleteEvent(newOptions)
+const rebindOnComplete = (oldOptions, newOptions) => {
```

Comment:
very minor: but you can do deconstructing on both options here.
`({onComplete: oldOnComplete, onComplete})`